ibmanalyticsengineapiv3/ibm_analytics_engine_api_v3_integration_test.go

```diff
@@ -47,11 +47,13 @@ var _ = Describe(`IbmAnalyticsEngineApiV3 Integration Tests`, func() {
 		ibmAnalyticsEngineApiService *ibmanalyticsengineapiv3.IbmAnalyticsEngineApiV3
 		serviceURL   string
 		config       map[string]string
+		// !!! Start of custom content to be copied !!!  
```

Comment:
lets add a small note on why these changes are required

Reply:
Added note on this